templates/mesh/odk2odm.html

```diff
@@ -17,7 +17,10 @@
 <div class="row">
     <div class="container">
         <h2>Mesh project '{{ model.name }}'</h2><span>Monitor ODK submissions, transfer submissions to an ODM task in
-        your ODM project, launch ODM tasks and monitor progress, transfer meshes to a game project.
+        your ODM project, launch ODM tasks and monitor progress, transfer meshes to a game project. Currently, direct transfer to game meshes is not yet supported.
+        Instead, please download your mesh from the Assets after creation, and go to <a href="https://github.com/localdevices/3DStreetView-game/blob/main/docs/build_unity_project.md">this</a>
+        link to prepare a Unity game with your mesh. Once that is done, zip your game or just the mesh file (extension .data), and upload it in this dashboard
+        in the <a href="/dashboard/game">Game project</a> menu. This step will be automated at a later stage.
```

Comment:
Working on it! I think meshlabserver will do the trick for a well-aligned mesh (that only needs 90degree rotation).